tests/phpunit/tests/login.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @group login
+ * @group login
+ */
+class Tests_Login extends WP_UnitTestCase {
+	function setUp() {
+		parent::setUp();
+		reset_phpmailer_instance();
+	}
+
+	function tearDown() {
+		reset_phpmailer_instance();
+		parent::tearDown();
+	}
+
+	/**
+	 * @runInSeparateProcess
+	 */
+     public function test_reset_password() {
+     	ob_start();
```

Comment:
These two lines should be indented with tabs instead of spaces. You can see the misalignment here in the GitHub web view, and I would also highly recommend configuring your editor to show you a visual difference between tabs and spaces.